docs/maui/views/DrawingView.md

```diff
@@ -0,0 +1,133 @@
+---
+title: DrawingView - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: "The DrawingView allows to draw lines, save the image and restore it by settings the list of lines."
+ms.date: 04/14/2022
+---
+
+# DrawingView
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `DrawingView` allows to draw lines, save the image and restore it by settings the list of lines.
+
+## Syntax
+
+The `DrawingView` is invoked using C#.
+
+### XAML
+
+```xml
+<views:DrawingView
+            x:Name="DrawingViewControl"
+            Lines="{Binding MyLines}"
+            IsMultiLineModeEnabled="true"
+            ShouldClearOnFinish="true"
+            DrawingLineCompletedCommand="{Binding DrawingLineCompletedCommand}"
+            DrawingLineCompleted="OnDrawingLineCompletedEvent"
+            LineColor="Red"
+            LineWidth="5"
+            BackgroundColor="DarkGray"
+            HorizontalOptions="FillAndExpand"
+            VerticalOptions="FillAndExpand" />
+```
```

Comment:
I really quite like exposing limited properties in a set of examples, for example we could show the basics to render a `DrawingView`, then an example on multi line support, then how to handle when a line is added.

What do you think? I am happy to try and expand on this if you prefer?

Reply:
I like the move towards smaller sets of examples. I wonder if we could also include the C# in each of those sections too. Well at least any C# that might relate to it. 

I broke out of the original format with the Syntax headings from other pages as showing these concepts off with some small steps felt right. The popup docs page shows what I mean by including the C# and XAML in small related sections with a small description of what it is doing: https://docs.microsoft.com/en-gb/dotnet/communitytoolkit/maui/views/popup